### DISCUSSION TITLE
Window position and title-bar button under XFCE

### DISCUSSION THREAD

**Discussion by AnonymousRonin** [lxqt/lxqt#2512] [General]
This is in Kali so XFCE4 with qt but the issue seems to be related to or affected by something. When I launch it from anywhere,, even it's file location, it pins itself in a corner of my monitor and is missing the close button.  It works except I can't use it.  I can however launch the root terminal which as you know is a qterminal and it works just fine.  Same with xfce terminal.  I have tried uninstalling that along with almost every qt app I had and reinstalling it to no avail.  I ran dmesg but I don't know what half that stuff means, it always looks scary to me.

**Comment by Gnomasz** [comment]
1. What is "that" that you launch?
2. What do you mean by "it works except I can't use it"?
3. Please provide a screenshot that shows the issue.

Please respect the readers. Take your time to clearly describe your issue, or no-one will bother trying to decipher what you mean.

**Comment by tsujan** [comment]
@AnonymousRonin 

As @Gnomasz correctly mentioned, it's good to be more specific when asking for help. "Completely lost" isn't a good title.

Placing windows and giving them title-bar buttons is the job of WM (window manager). First check which WM you use. Most probably it is xfwm4, because you said you work under XFCE4. Second, some WMs have extra rules for putting specific windows at certain positions or changing their title-bars. So, check whether someone has added such a rule.

Also, you may want to ask for help in Kali forums to see if other Kali users have encountered similar issues. Some distros may do weird things or have temporary problems (broken upgrades, bad default settings, etc.), and their users may know how to deal with them.

> I have tried uninstalling that along with almost every qt app I had and reinstalling it to no avail.

Unlike MS Windows, in Linux it isn't re-installation but upgrading that may solve a series of problems, although many problems may not be fixed in that way: you need to find what has caused them.
